extensions/default/src/utils/getDirectURL.js

```diff
@@ -57,31 +51,19 @@ const getDirectURL = (config, params) => {
   const BulkDataURI =
     (value && value.BulkDataURI) ||
     `series/${SeriesInstanceUID}/instances/${SOPInstanceUID}${defaultPath}`;
-  const hasQuery = BulkDataURI.indexOf('?') != -1;
-  const hasAccept = BulkDataURI.indexOf('accept=') != -1;
+  const hasQuery = BulkDataURI.indexOf('?') !== -1;
+  const hasAccept = BulkDataURI.indexOf('accept=') !== -1;
   const acceptUri =
     BulkDataURI +
     (hasAccept ? '' : (hasQuery ? '&' : '?') + `accept=${defaultType}`);
-  if (BulkDataURI.indexOf('http') === 0) {
-    if (tag === 'PixelData' || tag === 'EncapsulatedDocument') {
-      return `${wadoRoot}/studies/${StudyInstanceUID}/series/${SeriesInstanceUID}/instances/${SOPInstanceUID}/rendered`;
-    } else {
-      return acceptUri;
-    }
-  }
-  if (BulkDataURI.indexOf('/') === 0) {
-    return wadoRoot + acceptUri;
-  }
-  if (BulkDataURI.indexOf('series/') == 0) {
-    return `${wadoRoot}/studies/${StudyInstanceUID}/${acceptUri}`;
-  }
-  if (BulkDataURI.indexOf('instances/') === 0) {
-    return `${wadoRoot}/studies/${StudyInstanceUID}/series/${SeriesInstanceUID}/${acceptUri}`;
-  }
-  if (BulkDataURI.indexOf('bulkdata/') === 0) {
-    return `${wadoRoot}/studies/${StudyInstanceUID}/${acceptUri}`;
+
+  if (tag === 'PixelData' || tag === 'EncapsulatedDocument') {
+    return `${wadoRoot}/studies/${StudyInstanceUID}/series/${SeriesInstanceUID}/instances/${SOPInstanceUID}/rendered`;
   }
-  throw new Error('BulkDataURI in unknown format:' + BulkDataURI);
+
+  // since we fixed the bulkDataURI to be absolute (in case of the relative),
+  // we can just return it here
+  return BulkDataURI;
```

Comment:
This should be the accept URI, because the DICOMweb standard states that the default is multipart related, and then separately states that the accept parameter is the URL parameter equivalent of the accept header.